tethysext/atcore/controllers/resource_workflows/map_workflows/spatial_condor_job_mwv.py

```diff
@@ -73,23 +73,6 @@ def process_step_options(self, request, session, context, resource, current_step
             next_step=next_step
         )
 
-    def get_tabular_data_for_previous_steps(self, current_step):
```

Comment:
The thing is that ResourceWorkflow is not a parent for SpatialCondorJobMWV. adding it just so we can have that functions seems a little overkill, no? Take a look at what else was already in the utilities before you decide.

Reply:
You are right, it is not a parent of the view because it is a model. ResourceWorkflow is the workflow model, which you will have access to, no? Look at the other methods on ResourceWorkflow. Seems like a good fit and it will be available anywhere you have a workflow.